### PR TITLE
Bump kotest to 5.6.2

### PR DESCRIPTION
**Pull request by glockbender**
closes: https://github.com/kotest/kotest-extensions-allure/issues/47

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,20 +1,26 @@
 [versions]
-kotest = "5.3.2"
-allure = "2.18.1"
+kotlin = "1.8.21"
+kotest = "5.5.5"
```
Comment (Kantis) [inline gradle/libs.versions.toml:3]: Did you run into issues with 5.6.2?

File: build.gradle.kts [hunk 2]
```diff
@@ -2,8 +2,8 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 
 plugins {
    id("kotest-publishing-conventions")
-   kotlin("jvm") version "1.6.21"
-   id("io.qameta.allure") version "2.10.0"
+   alias(libs.plugins.kotlinJvm).apply(true)
+   alias(libs.plugins.allure).apply(true)
```
Comment (Kantis) [inline build.gradle.kts:6]: `.apply(true)`  is done by default, can be omitted

**Review by Kantis — COMMENTED** [review]
Overall I think the changes make sense. I'd be OK with merging this if you're willing to try out the snapshots and report back if it's working well. I don't personally use Allure so I can't tell for sure if it's all good or not.

**Review by Kantis — COMMENTED** [review]
Overall I think the changes make sense. I'd be OK with merging this if you're willing to try out the snapshots and report back if it's working well. I don't personally use Allure so I can't tell for sure if it's all good or not.

File: build.gradle.kts (hunk 2 above)
Comment (glockbender) [reply build.gradle.kts:6]: done

File: gradle/libs.versions.toml (hunk 1 above)
Comment (glockbender) [reply gradle/libs.versions.toml:3]: Nope. Bumped

**Review by Kantis — APPROVED** [review]

**Comment by Kantis** [comment]
Let me know how the snapshot works for you. If all is good we can add some compatibility info to the readme and do a release.

**Pull request merged**